src/generated/resources/assets/dungeonsdelight/lang/zh_cn.json

```
@@ -0,0 +1,402 @@
+  "item.dungeonsdelight.fried_ghast_calamari": "炸恶魂触须圈",
+  "item.dungeonsdelight.gelled_salad": "凝胶沙拉",
+  "item.dungeonsdelight.ghast_calamari": "恶魂触须圈",
+  "item.dungeonsdelight.ghast_roll": "恶魂寿司",
+  "item.dungeonsdelight.ghast_tentacle": "恶魂触须",
+  "item.dungeonsdelight.ghastly_spirits": "恶魂烈酒",
+  "item.dungeonsdelight.ghoulash": "怪拉什",
+  "item.dungeonsdelight.glow_berry_gelatin": "碗装发光浆果凝胶",
+  "item.dungeonsdelight.golden_cleaver": "金斩切刀",
+  "item.dungeonsdelight.gritty_flesh": "沙砾肉",
+  "item.dungeonsdelight.guardian_angel": "盘装守护天使",
+  "item.dungeonsdelight.gunk": "黏胶块",
+  "item.dungeonsdelight.gunk_arrow": "黏胶箭",
+  "item.dungeonsdelight.gyudon": "圾丼",
+  "item.dungeonsdelight.hydra_fricassee": "白汁九头蛇肉块",
+  "item.dungeonsdelight.iron_cleaver": "铁斩切刀",
+  "item.dungeonsdelight.liveroot_beer": "活根啤酒",
+  "item.dungeonsdelight.logo_item": "标志物品",
+  "item.dungeonsdelight.malicious_sandwich": "恶意三明治",
+  "item.dungeonsdelight.maze_roll": "迷宫卷",
+  "item.dungeonsdelight.meef_wellington": "惠灵顿牛头人肉排",
+  "item.dungeonsdelight.monster_burger": "怪物汉堡",
+  "item.dungeonsdelight.monster_cake_slice": "怪物蛋糕切片",
+  "item.dungeonsdelight.monster_muffin": "怪物松饼",
+  "item.dungeonsdelight.monster_yam_spawn_egg": "怪物山芋刷怪蛋",
+  "item.dungeonsdelight.netherite_cleaver": "下界合金斩切刀",
+  "item.dungeonsdelight.ominous_omelette": "不祥欧姆蛋",
+  "item.dungeonsdelight.ossobusco": "碗装红烩人小膝",
+  "item.dungeonsdelight.poi": "芋泥",
+  "item.dungeonsdelight.poisonous_poutine": "有毒肉汁奶酪薯条",
+  "item.dungeonsdelight.rancid_reduction": "腐臭浓缩液",
+  "item.dungeonsdelight.rotbulb": "腐球茎",
+  "item.dungeonsdelight.rotten_tripe": "腐烂内脏",
+  "item.dungeonsdelight.salt_soaked_stew": "盐水炖菜",
+  "item.dungeonsdelight.scaly_fiddlehead_risotto": "鳞片暮色蕨烩饭",
+  "item.dungeonsdelight.sculk_apple": "幽匿苹果",
+  "item.dungeonsdelight.sculk_dogapple": "幽匿狗苹果",
+  "item.dungeonsdelight.sculk_mayo": "幽匿蛋黄酱",
+  "item.dungeonsdelight.sculk_polyp": "幽匿息肉",
+  "item.dungeonsdelight.sculk_tart_slice": "幽匿馅饼切片",
+  "item.dungeonsdelight.shiokara": "盐辛",
+  "item.dungeonsdelight.silverfish_fried_rice": "蠹虫炒饭",
+  "item.dungeonsdelight.silverfish_thorax": "蠹虫胸肉",
+  "item.dungeonsdelight.sinigang": "酸腻羹",
+  "item.dungeonsdelight.slicorice": "甘草糖",
+  "item.dungeonsdelight.slime_bar": "粘液棒",
+  "item.dungeonsdelight.slime_noodles": "粘液面条",
+  "item.dungeonsdelight.smoked_spider_meat": "烟熏蜘蛛肉",
+  "item.dungeonsdelight.sniffer_shank": "生嗅探兽腿肉",
+  "item.dungeonsdelight.snifferwurst": "生嗅探兽德肠",
+  "item.dungeonsdelight.snuffledog": "嗅探兽热狗",
+  "item.dungeonsdelight.soaked_skewer": "浸泡肉串",
+  "item.dungeonsdelight.soft_serve_sniffer_egg": "嗅探兽蛋霜淇淋",
+  "item.dungeonsdelight.spider_extract": "蜘蛛提取物",
+  "item.dungeonsdelight.spider_eye_salmagundi": "蜘蛛眼杂烩",
+  "item.dungeonsdelight.spider_meat": "蜘蛛肉",
+  "item.dungeonsdelight.spider_pie_slice": "蜘蛛派切片",
+  "item.dungeonsdelight.spider_tanghulu": "蜘蛛眼糖葫芦",
+  "item.dungeonsdelight.stained_scrap": "污秽残片",
```

```suggestion
  "item.dungeonsdelight.stained_scrap": "污秽碎片",
```
与下界合金碎片（Netherite Scrap）保持一致。